Langs/040B.lang

```diff
@@ -4,56 +4,56 @@ Translator=NabsiYa
 Version=2.3.0.1
 [Strings]
 Donate=Lahjoita
-Check for Updates=Etsi Päivityksiä
-Your Windows 11 Compatibility Results are Below=Sinun Yhteensopivuus Tulokset Windows 11 Kanssa Ovat Alapuoella
+Check for Updates=Etsi päivityksiä
+Your Windows 11 Compatibility Results are Below=Windows 11 -yhteensopivuustuloksesi ovat alapuolella
 Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Nyt voit löytää WhyNotWin11:n osoitteesta https://www.whynotwin11.org/
-Results Based on Currently Known Requirements!=Tulokset Ovat Pohjaisin Tällä Hetkellä Saaduista Vaatimuksista
+Results Based on Currently Known Requirements!=Tulokset pohjautuvat tämänhetkisiin tiedettyihin vaatimuksiin!
 Translation by=Nabsi#0001
-Architecture (CPU + OS)=Arkkitehtuuri (Processori + Käyttöjärjestelmä)
-Boot Method=Käynnistys Tapa
-CPU Compatibility=Processorin Yhteensopivuus
-CPU Core Count=Processorin Ytimien Määrä
-CPU Frequency=Processori Kellotaajuus
+Architecture (CPU + OS)=Arkkitehtuuri (suoritin + käyttöjärjestelmä)
+Boot Method=Käynnistystapa
+CPU Compatibility=Suorittimen yhteensopivuus
+CPU Core Count=Suorittimen ytimien määrä
+CPU Frequency=Suorittimen kellotaajuus
 DirectX + WDDM2=DirectX + WDDM2
-Disk Partition Type=Levyn Osio Tyyppi
-RAM Installed=Ramia Asennettuna
-Secure Boot=Turvallinen Käynnistys
-Storage Available=Tilaa Vapaana
-TPM Version=TPM versio
+Disk Partition Type=Levyn osiotyyppi
+RAM Installed=Keskusmuistia asennettuna
+Secure Boot=Turvallinen käynnistys
+Storage Available=Tilaa vapaana
+TPM Version=TPM-versio
 Checking...=Varmistetaan...
-64 Bit CPU=64-Bit:n Processori
-64 Bit OS=64-Bit:n Käyttöjärjestelmä
-32 Bit CPU=32-Bit:n Processori
-32 Bit OS=32-Bit:n Käyttöjärjestelmä
-Unable to Determine=Ei Voida Määrittää
-Unable to Check List=Ei Voida Katsoa Listaa
-Error Accessing List=Ongelma Avatessa Listaa
-Not Currently Listed as Compatible=Ei Tällä Hetkellä Tuoettu
-Listed as Compatible=Listattu Tuetuksi
+64 Bit CPU=64-bittinen suoritin

```

Comment:
also i wanna mention specially that i was told this was "incorrect" so i used "Bit:n" instead which annoys me upto this day so thanks for correcting it.

Reply:
I didn't mean that "bittinen" was incorrect, I meant that you used x64 and x32, sorry for not being clearer.